cpp/src/gandiva/function_registry.h

```diff
@@ -33,6 +33,9 @@ class GANDIVA_EXPORT FunctionRegistry {
   /// Lookup a pre-compiled function by its signature.
   const NativeFunction* LookupSignature(const FunctionSignature& signature) const;
 
+  /// \brief register a new function into the function registry
+  static Status AddFunction(NativeFunction func);
```

Comment:
It seems that `Function` is redundant in `AddFunction()` because this class is `FunctionRegistry`.

Reply:
+1. It was me who suggested the name `AddFunction`, now I agree that simply `Add` is enough.

Reply:
Fixed